examples/single_fov_ome_zarr.py

```diff
@@ -11,22 +11,23 @@
 import numpy as np
 
 from iohub.ngff import open_ome_zarr
+from datetime import datetime
 
 # %%
 # Set storage path
-
-store_path = f"{tempfile.gettempdir()}/ome.zarr"
+timestamp = datetime.now().strftime('%d%H%M%S')
+store_path = '/home/eduardo.hirata/Documents/tmp/' +timestamp+ "ome.zarr"
```

Comment:
We shouldn't hardcode any specific path in the repo. I suggest `os.path.expanduser('~/tmp/...')`.

Reply:
ups didn't mean to push and alter the example code. Ill remove this one. Good catch.